pytensor/tensor/blas.py

```diff
@@ -210,14 +197,11 @@ def make_node(self, y, alpha, A, x, beta):
 
     def perform(self, node, inputs, out_storage):
         y, alpha, A, x, beta = inputs
-        if (
-            have_fblas
-            and y.shape[0] != 0
-            and x.shape[0] != 0
-            and y.dtype in _blas_gemv_fns
-        ):
-            gemv = _blas_gemv_fns[y.dtype]
-
+        try:
+            gemv = get_blas_funcs(names="gemv", dtype=y.dtype)
```

Comment:
I suppose we want to cache this so we are not calling it every iteration of perform. Elemwise does something like that for the respective ufunc

Otherwise does cloudpickle work if we were using this in the old dict instead?

Reply:
I'll change this because it wasn't necessary for the fix. I thought that the bug was coming from some weird `cloudpickle` magic that pulled in variables from the global scope, but the problem was actually `ScipyGer` that stores a fortran object in its tag.